packages/scheduled-tasks/src/lib/ScheduledTaskHandler.ts

```diff
@@ -223,7 +223,9 @@ export class ScheduledTaskHandler {
 
 		result.inspectErr((error) => container.client.emit(ScheduledTaskEvents.ScheduledTaskError, error, piece, payload));
 
-		const value = result.unwrapOr(null);
+		const value = result.unwrapOrElse((error) => {
+			throw error;
+		});
```

Comment:
```suggestion
		const value = result.match({
			ok: (value) => value,
			err(error) {
				container.client.emit(ScheduledTaskEvents.ScheduledTaskError, error, piece, payload);
				throw error;
			}
		});
```

I think this should also work?